### PR TITLE
Configuration, documentation, and cleanup

### PR DESCRIPTION
**Pull request by pothos**
- Handle request length option as list
- Allow to disable TCP, HTTP, or gRPC benchmarking
- Update default values and documentation
- Remove HTTP/gRPC info from TCP graph labels

### CODE REVIEW

**Comment by pothos** [comment]
Also fixed that the benchmark script hangs when the iperf connection fails.

File: profiling/benchmark-cargo-test-fortio.sh [hunk 1]
```diff
@@ -18,74 +22,105 @@ BRANCH_NAME=$(git symbolic-ref -q HEAD)
...
   else
     RPS="$HTTP_RPS"
     XARG=""
     if [ "$MODE" = "gRPC" ]; then
       RPS="$GRPC_RPS"
       XARG="-grpc -s $GRPC_STREAMS"
     fi
-    for r in $RPS; do
+    for l in $REQ_BODY_LEN; do
+     for r in $RPS; do
```
Comment (dongsupark) [inline profiling/benchmark-cargo-test-fortio.sh]: NIT: indent with 2 spaces?

**Review by dongsupark — APPROVED** [review]
Looks good.
See below:

**Comment by pothos** [comment]
Ported the failure handling to the wrk2/strest-grpc script but not the output hiding.

**Pull request merged**